Signal/src/Storyboard/Storyboard.storyboard

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I am not sure how storyboard changes got in, maybe due to rebasing on master. these should not be part of the PR